### ISSUE TITLE
Is Focal Loss correct ??

### ISSUE THREAD

**Issue by LifeBeyondExpectations** [marvis/pytorch-yolo2#114]
This is the origin code of Focal Loss : [focal_loss_URL](https://github.com/pytorch/examples/blob/master/mnist/main.py)

It uses sigmoid but the focal loss in this repository use softmax .. I am not sure about it.

**Comment by yuyijie1995** [comment]
Can you tell me how to add the focal loss code into the original model?
